### PR TITLE
fix regression: update thelper to v0.2.1

### PR DESCRIPTION
**Pull request by ldez**
diff: https://github.com/kulti/thelper/compare/v0.2.0...v0.2.1

With v0.2.0, all the sub-test functions are detected as a problem.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -234,6 +234,8 @@ github.com/kr/text v0.2.0 h1:5Nx0Ya0ZqY2ygV366QzturHI13Jq95ApcVaJBhpS+AY=
 github.com/kr/text v0.2.0/go.mod h1:eLer722TekiGuMkidMxC/pM04lWEeraHUUmBw8l2grE=
 github.com/kulti/thelper v0.2.0 h1:0hWMFWREtbyuGTPl/d2c1Kq/VfzNM97NzRwp1zMLaAc=
 github.com/kulti/thelper v0.2.0/go.mod h1:vMu2Cizjy/grP+jmsvOFDx1kYP6+PD1lqg4Yu5exl2U=
+github.com/kulti/thelper v0.2.1 h1:H4rSHiB3ALx//SXr+k9OPqKoOw2cAZpIQwVNH1RL5T4=
```
Comment (SVilgelm) [inline go.sum:237]: please run `go mod tidy` it should clean up the `go.sum` from `thelper v0.2.0` lines

**Review by SVilgelm — CHANGES_REQUESTED** [review]

File: go.sum (hunk 1 above)
Comment (ldez) [reply go.sum:235]: Fixed

**Review by kulti — APPROVED** [review]
LGTM

File: test/testdata/thelper.go [hunk 2]
```diff
@@ -28,3 +28,22 @@ func bhelperWithNotFirst(s string, b *testing.B, i int) { // ERROR "parameter \*
 func bhelperWithIncorrectName(o *testing.B) { // ERROR "parameter \*testing.B should have name b"
 	o.Helper()
 }
+
+func TestTHelper(t *testing.T) {
```
Comment (kulti) [inline test/testdata/thelper.go:32]: How about a more specific name? E.g. `TestSubtestIsNotChecked` or `TestSubtestShouldNotBeChecked`.

Comment (ldez) [reply test/testdata/thelper.go]: Fixed

**Pull request merged**